src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/config/ApiExceptionHandler.kt

```diff
@@ -34,7 +36,7 @@ class RestConfiguration {
   }
 
   @ExceptionHandler(BusinessException::class)
```

Comment:
Shouldn't this be ServiceUnavailableException rather than BusinessException? 
Also we still need the handleBusinessException